open_schools_platform/organization_management/organizations/urls.py

```diff
@@ -20,7 +20,4 @@
     path('/<uuid:pk>/students/export', OrganizationStudentProfilesExportApi.as_view(),
          name='export-organization-students'),
     path('/<uuid:pk>/analytics', GetAnalytics.as_view(), name='analytics'),
-    path('/<uuid:organization_id>/student-profile/<uuid:student_profile_id>/circles', QueriesCirclesOrganizationStudent.as_view(), name='queries-organization-student-profile'),
-    path('/<uuid:pk>/teachers', OrganizationTeachersListApi.as_view(), name='teachers-list'),
-    path('/teachers/<uuid:pk>', GetTeacherApi.as_view(), name='get-teacher')
-]
```

Comment:
If you remove endpoints, you should either change or remove the tests for these URLs.

I'm out of context, is it really necessary to remove those endpoints?

Reply:
I didn't write it and was afraid that I would accidentally appropriate it. do I need to leave this code and tests?